### PR TITLE
Add accumulate exercise

### PR DESCRIPTION
**Pull request by hiljusti**

### CODE REVIEW

File: exercises/practice/accumulate/accumulate.factor [hunk 1]
```diff
@@ -0,0 +1,6 @@
+USING: kernel ;
+IN: accumulate
+
+: accumulate ( seq xform: ( a -- b ) -- newseq )
```
Comment (hiljusti) [inline exercises/practice/accumulate/accumulate.factor:4]: 1. Update this word name to be `accum` (the tests require it right now, and

File: exercises/practice/accumulate/.docs/instructions.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+And the operation:
+
+- square a number (`[ sq ]`)
+
+Your code should be able to produce the collection of squares:
+
+- 1, 4, 9, 16, 25
+
+Check out the test suite to see the expected function signature.
```
Comment (hiljusti) [inline exercises/practice/accumulate/.docs/instructions.md:20]: This line is a copy-paste error (Thanks @kotp for this)

The "function signature" ([stack effect declaration](https://docs.factorcode.org/content/article-effects.html)) is already provided in `accumulate.factor`

**Review by kotp — APPROVED** [review]

**Comment by kotp** [comment]
Are you attending to the config file as well.  The confliglet check is reporting issues with that.

**Comment by hiljusti** [comment]
@kotp Yeah, I just now merged changes that should fix the config file: https://github.com/exercism/factor/commit/f08982b2db282fa397e98d837e7312906232add9

**Comment by hiljusti** [comment]
Just rebased and also moved the example code to `.meta` based on a suggestion from Erik.

**Review by ErikSchierboom — COMMENTED** [review]
@hiljusti You also need to add an entry to the root `config.json` file. See https://github.com/exercism/docs/blob/main/building/tracks/config-json.md#practice-exercises

Note that the uuid should be a unique generated UUID v4. The easiest way to do that is to use `configlet uuid`

**Comment by hiljusti** [comment]
I'll update to include the entry.

I obviously have skipped a lot of documentation... - I didn't even know that `configlet` thing was an application I could run locally. (Note for any future viewers, I found it [here](https://github.com/exercism/configlet)) Configlet has a snazzy sync command for keeping fresh on exercises and a config linter which are awesome. I don't think I would use it for uuid when I already have `uuidgen` (I'm on Linux, but it will be pre-installed on MacOS & BSD distros too)

**Comment by ErikSchierboom** [comment]
>  I don't think I would use it for uuid when I already have uuidgen

Which is totally fine, as long as it generates a v4 UUID.

**Review by ErikSchierboom — APPROVED** [review]

File: config.json [hunk 3]
```diff
@@ -86,6 +86,19 @@
...
+        "slug": "accumulate",
+        "name": "Accumulate",
+        "uuid": "8a373305-98c9-4b19-9011-085abe19cfed",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5,
+        "topics": [
+          "quotations",
+          "collections",
+          "combinators"
```
Comment (ErikSchierboom) [inline config.json:100]: Topics were used in existing, v2 tracks. This field has been superseded by `practices` and `prerequisites`. See https://github.com/exercism/docs/blob/main/building/tracks/config-json.md#practice-exercises

**Review by ErikSchierboom — APPROVED** [review]

File: config.json (hunk 3 above)
Comment (hiljusti) [reply config.json:100]: Interesting - I'll update. Is the general idea in v3 that progression is less linear?

**Pull request merged**